packages/denylist/src/denylist.js

```diff
@@ -47,3 +47,40 @@ export const denylistGet = async function (request, env) {
   })
   return response
 }
+
+/**
+ * Batch denylist check. POST an Array of cid strings.
+ * Returns the subset of the Array that are on the deny list.
+ * @param {import('itty-router').Request} request
+ * @param {import('./env').Env} env
+ */
+export async function denylistPost (request, env) {
+  if (!request.json) {
```

Comment:
AFAIK this is not undefined if content-type is not json...

Reply:
ah yes, right, i was following the itty-route request type which defines .json as optional, but isn't want i want to test for here. Fixing!